scripts/run_spec_plan_kimi.py

```
@@ -772,7 +867,7 @@ def build_tasks_prompt(packed_xml: str, ticket_text: str, specs_content: str, pl
     • Visually inspect and analyze screenshots for: MUI v9 theme palette compliance (no hardcoded hex/rgb), typography hierarchy, padding/margin rhythm, text truncation/overflows, RTL mirroring alignments, dark/light contrast
     • Iterative self-loop: inspect screenshot -> identify UI defect -> patch MUI `sx` tokens -> re-capture screenshot -> repeat until visually polished
   - [ ] X.Y.SR **Semantic Review**: Verify zero direct style props (sx only), no hardcoded strings/colors, useAppTranslation property access, *Outlined icons
-  - [ ] X.Y.IV **Instruction Verification**: Validate against frontend.instructions.md, mobile-desktop.instructions.md, and layer AGENTS.md
+  - [ ] X.Y.IV **Instruction Verification**: Validate against `.agents/instructions/frontend.instructions.md` + layer AGENTS.md (the ONLY instruction files are `.agents/instructions/{frontend,backend,tests}.instructions.md` — nothing else exists)
```

Ruff (0.16.2)

[error] Undefined name `frontend`

(F821)